NickvisionMoney.Shared/Controllers/MainWindowController.cs

```diff
@@ -382,8 +382,8 @@ public void RemoveRecentAccount(RecentAccount recentAccount)
     private async void OnTransferSent(object? sender, Transfer transfer)
     {
         await AddAccountAsync(transfer.DestinationAccountPath, false, transfer.DestinationAccountPassword);
+        await Task.Delay(500);
```

Comment:
I remember you once complained about using delays as workarounds making the app slower :D And @JoseBritto confirmed my fear that 500 can be not enough. Maybe add something like `await controller.EnsureStartup();` that will not return until startup is finished (similar to what I suggested in previous PR)?

Reply:
done in latest commit.